doc/versionlock.rst

```diff
@@ -117,6 +117,13 @@ Options
 
 All general DNF options are accepted, see `Options` in :manpage:`dnf(8)` for details.
 
+``--raw``
+    Do not parse ``<package-name-spec>`` to NEVRAs to find specific version to lock to. Instead ``<package-name-spec>`` are used as they are. This enables locking to not yet available versions of the package.
```

Comment:
Okay, one more minor thing, can you please replace _"parse"_ with _"resolve"_ here?
> Do not ** resolve** ``<package-name-spec>`` to NEVRAs ...

I think it better describes what's going to happen.